### PR TITLE
obs: Enable ubuntu 20.04

### LINKED ISSUE #1080: pkgs: Enable ubuntu 20.04
**Which feature do you think can be improved?**

Enable ubuntu 20.04

**How can it be improved?**
1. Send a PR to modify the following file:
https://github.com/kata-containers/packaging/blob/master/obs-packaging/distros_x86_64
The change should include 
```diff
+xUbuntu_20.04::Ubuntu:20.04::universe
```
2. Run the packaging CI to verify it builds 
`/AzurePipelines run`
3. Even if passed the CI, verify that the packages were created successfully
https://build.opensuse.org/project/show/home:katacontainers:ci:x86_64:packaging-PR-<YOUR_PR_NUMBER>
4. The CI for packaging only tries to  validate if the build was successful, verify in a ubuntu 20.04 system, if it is working 
```bash
$ ARCH=$(arch)
$ YOUR_PR_NUMBER="<INSERT-PR-NUMER-HERE>"
$ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:katacontainers:ci:x86_64:packaging-PR-${YOUR_PR_NUMBER}/xUbuntu_$(lsb_release -rs)/ /' > /etc/apt/sources.list.d/kata-containers.list"
$ curl -sL  http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/${BRANCH}/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
$ sudo -E apt-get update
$ sudo -E apt-get -y install kata-runtime kata-proxy kata-shim
```

5. All ready to merge  :) 

**Additional Information**

Anything else to add?

**Before raising this feature request**

Have you looked at the [limitations document](https://github.com/kata-containers/documentation/blob/master/Limitations.md)?

### PR DESCRIPTION
**Pull request by GabyCT**
This PR enables the obs packages for ubuntu 20.04.

Fixes #1080

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
@jcvenegas it looks like obs packages for ubuntu 20.04 passed, however, this is failing but because of opensuse https://build.opensuse.org/package/live_build_log/home:katacontainers:ci:x86_64:packaging-PR-1081/linux-container/openSUSE_Tumbleweed/x86_64

File: obs-packaging/distros_x86_64 [hunk 1]
```diff
@@ -17,3 +17,4 @@ openSUSE_Leap_15.1::openSUSE:Leap:15.1::standard
 openSUSE_Tumbleweed::openSUSE:Factory::snapshot
 xUbuntu_16.04::Ubuntu:16.04::universe,universe-update,update
 xUbuntu_18.04::Ubuntu:18.04::universe
+xUbuntu_20.04::Ubuntu:18.04::universe
```
Comment (jodh-intel) [inline obs-packaging/distros_x86_64:20]: Shouldn't this be...?

```
xUbuntu_20.04::Ubuntu:20.04::universe
```

Comment (GabyCT) [reply obs-packaging/distros_x86_64]: @jodh-intel changes applied

**Pull request merged**